configs/kubernetes-rules-configmap.yaml

```diff
@@ -846,3 +846,79 @@ data:
           branch: release-1.12
         - repository: client-go
           branch: release-9.0
+    - destination: sample-cli-plugin
+      library: true
```

Comment:
is this a library? I.e. do we want complete Godeps? For libraries we remove some of the k8s.io dependencies to be go-gettable. Ugly but necessary.

Reply:
After looking into the staging repo, this is most certainly not a library, but a command only.

Reply:
so i change this to `library: false`?

Reply:
Fixed. set to false